README.md

```diff
@@ -15,6 +15,61 @@ def deps do
 end
 ```
 
+## Usage
+
+Validate data against a defined schema.
+
+* `Litmus.validate/2`
+
+If the data is valid, the function returns {:ok, data}. The data returned will be coerced according to the schema defined.
+
+If the data passed does not follow the rules defined in the schema, the function returns {:error, error_message}. It will also return error, in case we have a field in data, whose rule is not defined in the schema.
```

Comment:
can you limit lines in the readme to 80 characters?

Reply:
surround the tuples with ` so you get mono-spaced code fonts

Reply:
The last sentence could be more clearn. Maybe something like: "It will also return an error the data contains parameters that are not specified in the schema."